aidb_utilities/aidb_setup/aidb_factory.py

```diff
@@ -1,64 +1,68 @@
-import importlib
 import pandas as pd
 
 from aidb.config.config_types import InferenceBinding
 from aidb.engine import Engine
+from aidb.inference.examples.llm_inference_service import LLMInference
 from aidb.vector_database.chroma_vector_database import ChromaVectorDatabase
 from aidb.vector_database.faiss_vector_database import FaissVectorDatabase
 from aidb.vector_database.weaviate_vector_database import WeaviateVectorDatabase
 from aidb.vector_database.tasti import Tasti
 
-
 def get_tasti_config(tasti_config):
   vector_database = {
     'FAISS': FaissVectorDatabase,
     'CHROMA': ChromaVectorDatabase,
     'WEAVIATE': WeaviateVectorDatabase
   }
-  vector_database_config = tasti_config.vector_database
-  vector_database_type = vector_database_config['vector_database_type'].upper()
+
+  vector_database_type = tasti_config['type'].upper()
   try:
-    user_vector_database = vector_database[vector_database_type](**vector_database_config['auth'])
+    user_vector_database = vector_database[vector_database_type](**tasti_config['auth'])
   except KeyError:
     raise ValueError(f'{vector_database_type} is not a supported type. We support FAISS, Chroma and Weaviate.')
-  tasti_index = Tasti(vector_database=user_vector_database, **tasti_config.tasti_engine)
+  tasti_index = Tasti(vector_database=user_vector_database, **tasti_config['tasti_engine'])
 
   selected_vector_id_df = None
 
-  if hasattr(tasti_config, 'vector_id_csv') and tasti_config.vector_id_csv is not None:
-    selected_vector_id_df = pd.read_csv(tasti_config.vector_id_csv)
+  if 'vector_id_csv' in tasti_config:
+    selected_vector_id_df = pd.read_csv(tasti_config['vector_id_csv'])
     if len(selected_vector_id_df.columns) != 1:
       raise Exception('Vector id csv file should contain one column for vector id')
     selected_vector_id_df.columns.values[0] = 'vector_id'
 
   return tasti_index, selected_vector_id_df
 
 
+def setup_inference(service_name, service, service_config):
```

Comment:
This dictionary provides a list of available inference service types for users to choose from. For example, after adding a local inference service, it will be added to this dictionary, allowing users to select it by using 'local'.